src/modules/blockchainApplication/explore/components/BlockchainApplicationSkeleton/BlockchainApplicationSkeleton.js

```diff
@@ -23,7 +22,7 @@ const BlockchainApplicationSkeletonRow = () => (
 
 const BlockchainApplicationSkeleton = () => (
```

Comment:
We have to get rid of this at some point. But the rest look very well. Good job!